packages/slate-plugins/src/elements/link/withLink.ts

```diff
@@ -46,9 +53,10 @@ const upsertLink = (
  * Paste a string inside a link element will edit its children text but not its url.
  *
  */
-export const withLink = (options?: WithLinkOptions) => <T extends ReactEditor>(
-  editor: T
-) => {
+export const withLink = (
+  options?: WithLinkOptions,
+  rangeBeforeOptions?: RangeBeforeOptions
```

Comment:
Please add this option inside `WithLinkOptions`

Reply:
done